### PR TITLE
Update DASTool to 1.1.4

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

This updates DAS_Tool to 1.1.4

- Sees version bump
- Removal of now unnecessary warning
- Update of output files
- Addition of 'renamed' auxilary module (FASTATOCONTIG2BIN)

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by sateeshperi — APPROVED** [review]
lgtm

File: modules/dastool/fastatocontig2bin/meta.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: Fasta of list of fasta files recommended to be gathered via with .collect() of bins
+      pattern: "*.{fa,fa.gz,fas,fas.gz,fna,fna.gz,fasta,fasta.gz}"
+  - binner:
+      type: val
+      description: Name of the binning software (optional)
```
Comment (d4straub) [inline modules/dastool/fastatocontig2bin/meta.yml:37]: This isnt in the input?!
```suggestion
```

File: modules/dastool/dastool/meta.yml [hunk 2]
```diff
@@ -34,18 +34,15 @@ input:
...
       description: Predicted proteins in prodigal fasta format (>scaffoldID_geneNo)
       pattern: "*.{fa.gz,fas.gz,fasta.gz}"
   - db_directory:
       type: file
       description: (optional) Directory of single copy gene database.
-  - search_engine:
-      type: val
-      description: Engine used for single copy gene identification. USEARCH is not supported due to it being proprietary [blast/diamond]
 
 output:
```
Comment (d4straub) [inline modules/dastool/dastool/meta.yml:47]: `bins` seems to be missing?

**Review by d4straub — APPROVED** [review]
Just small meta changes required

**Pull request merged**